nixos/modules/services/games/factorio.nix

```diff
@@ -35,10 +35,10 @@ let
     auto_pause = true;
     only_admins_can_pause_the_game = true;
     autosave_only_on_server = true;
-    admins = [];
     non_blocking_saving = cfg.nonBlockingSaving;
   } // cfg.extraSettings;
   serverSettingsFile = pkgs.writeText "server-settings.json" (builtins.toJSON (filterAttrsRecursive (n: v: v != null) serverSettings));
+  serverAdminsFile = pkgs.writeText "server-admins.json" (builtins.toJSON cfg.admins);
```

Comment:
You are sure it is not server-adminlist.json?

Reply:
it's kind of irrelevant because it gets passed an argument when invoking the daemon. But I can change it if you prefer the canonical name

Reply:
I would prefer the canonical name because otherwise guides may no longer apply and make it harder to convert your server to nixos or setup a new one.

Reply:
done